### PR TITLE
Improvements for an edge case

### PR DESCRIPTION
**Pull request by Carbenium**

### CODE REVIEW

File: templates/aliases.j2 [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Managed by Ansible (role: mailclient), do not edit!
 {% for item in mailclient_to_redirect %}
-{{ item }}: {{ mailclient_system_owner }}@{{ mailclient_domain }}
+{{ item }}: {{ mailclient_system_owner }}
```
Comment (uubk) [inline templates/aliases.j2:6]: Can we somehow make this backwards-compatible? Quite a few deployments break if we do this.

**Review by uubk — CHANGES_REQUESTED** [review]

File: templates/aliases.j2 (hunk 1 above)
Comment (Carbenium) [reply templates/aliases.j2:6]: Uff. New variable? But that would make it really ugly...
I'm not sure if backwards compatibility is worth the hassle.

Comment (uubk) [reply templates/aliases.j2:6]: Hmm - how about a flag `mailclient_system_owner_include_domain` which defaults to true?

Comment (Carbenium) [reply templates/aliases.j2:6]: See 0fdb7da. I don't really like it, but that's me :stuck_out_tongue:

**Review by uubk — APPROVED** [review]
LGTM! Sorry for the long wait, and thank you for making the adjustment!

**Pull request merged**